.eslintrc.js

```diff
@@ -1,29 +1,21 @@
 module.exports = {
   root: true,
 
-  extends: [
-    '@metamask/eslint-config',
-    '@metamask/eslint-config/config/mocha',
-    '@metamask/eslint-config/config/nodejs',
-  ],
-
-  parser: 'babel-eslint',
-
   parserOptions: {
-    ecmaVersion: 2017,
-    ecmaFeatures: {
-      arrowFunctions: true,
-      classes: true,
-      experimentalObjectRestSpread: true,
-    },
+    ecmaVersion: 2018,
```

Comment:
We should use ECMAScript 2017 for now, until all of our supported browsers support ECMASCript 2018. At the moment 2017 is our limit.
```suggestion
    ecmaVersion: 2017,
```

Reply:
[Our supported browsers are listed here](https://github.com/MetaMask/metamask-extension/blob/develop/babel.config.js):
>'chrome >= 58',
>'firefox >= 56.2',

It looks like there are zero ECMAScript 2018 features supported by both these versions.

Reply:
I only changed it to `2018` to get rid of the `babel-eslint` dependencies. I don't understand why browser support is relevant considering that the package consumer has to build for their target environment anyway.

Reply:
We can still get rid of the `babel-eslint` dependencies and use 2017 here; I'm not sure what you mean exactly.

>I don't understand why browser support is relevant considering that the package consumer has to build for their target environment anyway.

This is not true; we don't currently transpile dependencies (with the exception of one, which we have to transpile because it uses Object rest/spread). We bundle them and minify them, but we don't run them through Babel.

Reply:
Oh I see, we actually are using one of these features (Object rest/spread). Well. We'll either need to keep Babel or remove usage of Object rest/spread.

Reply:
The two usages are trivially replaced with `Object.assign`, so that's probably the best path forward.

Reply:
In that case, we started shipping the spread operator with #33 😬

Reply:
Oh, which is actually fine: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax

Reply:
Oh wait, _sort of_ fine; it effectively raised our de-facto supported Chrome version from `58` to `60`.

Reply:
Unfortunately I do not think that is OK. That looks like it falls under ["Spread in object literals"](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax#Spread_in_object_literals), which means this would break for users of Chrome 58.x and 59.x

Reply:
Alright, I fixed this in 6e5916c, but I suspect nobody has been able to use MetaMask with Chrome <60 since #33 was merged.